inlong-manager/manager-web/src/main/java/org/apache/inlong/manager/web/controller/InlongClusterController.java

```diff
@@ -47,14 +50,53 @@
  * Inlong cluster controller
  */
 @RestController
-@RequestMapping("/cluster")
```

Comment:
why remove path identify in controller

Reply:
Put them in the under APIs, which will be searched conveniently.